src/newbank/server/NewBankClientHandler.java

```diff
@@ -30,34 +30,76 @@ public NewBankClientHandler(Socket s) throws IOException {
-                String password = in.readLine();
-                out.println("Checking Details...");
-                // authenticate user and get customer ID token from bank for use in subsequent requests
-                customer = bank.checkLogInDetails(userName, password);
-                if (customer != null) {
-                    break;
+        out.println("1. Existing user. 2. New user");
+        try{
+            String ans = in.readLine();
+
+            if(ans.equals("1")){
+            try {
+                CustomerID customer;
+                while (true) {
+                    // ask for user name
+                    out.println("Enter Username");
+                    String userName = in.readLine();
+                    // ask for password
+                    out.println("Enter Password");
+                    String password = in.readLine();
+                    out.println("Checking Details...");
+                    // authenticate user and get customer ID token from bank for use in subsequent requests
+                    customer = bank.checkLogInDetails(userName, password);
+                    if (customer != null) {
+                        break;
+                    }
+                    out.println("Log In Failed. Please try again:");
+                }
+                // if the user is authenticated then get requests from the user and process them
+                out.println("Log In Successful. What do you want to do?");
+                while (true) {
+                    String request = in.readLine();
+                    System.out.println("Request from " + customer.getKey());
+                    String response = bank.processRequest(customer, request);
+                    out.println(response);
                 }
-                out.println("Log In Failed. Please try again:");
+            } catch (IOException e) {
+                e.printStackTrace();
+             }
             }
-            // if the user is authenticated then get requests from the user and process them
-            out.println("Log In Successful. What do you want to do?");
-            while (true) {
-                String request = in.readLine();
-                System.out.println("Request from " + customer.getKey());
-                String response = bank.processRequest(customer, request);
-                out.println(response);
+            if(ans.equals("2")){
+                out.println("Set up your Username:");
+                String userName = in.readLine();
+                String passWord;
+                while(true){
+                  out.println("Set up your password:");
+                  String passWord1 = in.readLine();
+                  String passWord2 = "";
+                  while (!bank.passwordComplexity(passWord1).equals("Success")){
+                      out.println(bank.passwordComplexity(passWord1));
+                      out.println("Confirm your password:");
```

Comment:
System message here should be "Set up your password", since the password being set up does not meet the requirement. "Confirm your password" implies typing the same password again according to me.
```suggestion
                      out.println("Set up your password:");
```